css/tabBar.css

```diff
@@ -192,32 +194,28 @@ body.dark-mode #overlay {
 /* buttons */
 
 .navbar-action-button {
-	padding: 0 1em;
+	padding: 0 0.5em;
 	height: 36px;
-	line-height: 36px !important;
+	margin-right: 0.5em;
+	margin-top: -1px;
```

Comment:
The new tab and task buttons seem a bit small to me; would it be possible to make them larger?